### PR TITLE
fix: allow multiple characters in shortcuts

### PR DESCRIPTION
**Pull request by Xazin**
This cuts down on code duplication by allowing multiple characters to trigger one shortcut event.

**Note:**
We can discuss how we should split the characters, currently I split the character string by `,` and remove all spaces. This removes ` ` as a possibility to be included in a multi character command. We can just as easily build a RegExp pattern to do the matching if we want. (Or simply just have each chararacter be evaluated individually like `abc` would be `a`, `b`, and `c`.

### CODE REVIEW

File: lib/src/service/keyboard_service.dart [hunk 1]
```diff
@@ -158,7 +156,18 @@ class _AppFlowyKeyboardState extends State<AppFlowyKeyboard>
...
+      }
+
+      return event.character == character;
+    }
+
+    return keybindings.containsKeyEvent(event);
   }
+
+  List<String> _charactersFromString(String character) =>
+      character.split(',').map((c) => c.replaceAll(' ', '')).toList();
```
Comment (LucasXu0) [inline lib/src/service/keyboard_service.dart:172]:
```dart
bool canRespondToRawKeyEvent(RawKeyEvent event) {
    final character = this.character;
    if (character != null && character.isNotEmpty) {
      final characters = character.split(',').map((e) => e.trim());
      return characters.contains(event.character);
    }
    return keybindings.containsKeyEvent(event);
  }
```

Comment (LucasXu0) [reply lib/src/service/keyboard_service.dart:172]: I think it's more clear.

**Comment by LucasXu0** [comment]
LGTM.

**Comment by Xazin** [comment]
@LucasXu0 I almost forgot about this, it's ready.

**Comment by LucasXu0** [comment]
would you mind re-opening a new PR to refactor the `CharacterShortcutEvent`?

**Comment by Xazin** [comment]
> would you mind re-opening a new PR to refactor the `CharacterShortcutEvent`?

I'll refactor it accordingly sometime soon. :+1:

**Comment by MichalNemec** [comment]
Hello, any news regarding this? i would like to have custom character shortcut for #whatever

**Pull request closed without merging**